perf/monitoring/datarepo-monitoring.yaml

```diff
@@ -0,0 +1,109 @@
+---
+create-secret-manager-secret:
+  enabled: true
+  secrets:
+    - secretName: grafana-google-client-secret
+      vals:
+        - kubeSecretKey: GF_AUTH_GOOGLE_CLIENT_SECRET
+          path: secret/dsde/datarepo/perf/clientid-secret.json
+          vaultKey: oauthClientSecret
+
+backend-config:
+  enabled: true
+
+grafana-gcp-managed-cert:
+  enabled: true
+  fullnameOverride: grafana-gcp-managed-cert
+  domainName: datarepo-grafana.datarepo-perf.broadinstitute.org
+
+prometheus-gcp-managed-cert:
+  enabled: true
+  fullnameOverride: prometheus-gcp-managed-cert
+  domainName: datarepo-prometheus.datarepo-perf.broadinstitute.org
+
+kube-prometheus-stack:
+  enabled: true
+  prometheusOperator:
+    admissionWebhooks:
+      enabled: false
+    createCustomResource: "false"
+  prometheus:
+    serviceAccount:
+      annotations:
+        iam.gke.io/gcp-service-account: prometheus-sa@broad-jade-perf.iam.gserviceaccount.com      # workloadID annotation for sidecar to write to stackdriver
```

Comment:
If nothing else, should we use this SA in dev?  Though it may want its own

Reply:
The SA are project specific, dev is not in alignment with perf,staging and such.